docs/contributing/coding-standards/components.md

```diff
@@ -10,19 +10,19 @@ Generally, folder and file names should be singular, for example ‘accordion’
 
 When creating your component, you should create the following files in the component’s folder:
 
-      - `README.md`
-      - `_[component-name].scss`
-      - `_index.scss`
-      - `[component-name].yaml`
-      - `macro.njk`
-      - `template.njk`
-      - `template.test.js`
+- `README.md` - Light documentation pointing to installation instructions and component documentation on <https://design-system.service.gov.uk/>
+- `_[component-name].scss` - A SCSS file to generate the styles just for the component, importing the variables, functions and mixins from `../../base` and the styles of the component (`./index.scss`).
+- `_index.scss` - The actual styles for the component, that can be imported either on their own by `[component-name].scss` or alongside other components in `components/_all.scss` (or by the consuming application)
+- `[component-name].yaml` - A list of the options expected by the component, used for listing in the documentation 
```

Comment:
This file also includes examples of the component with specific options. These are used to generate examples in the documentation, as well as to run tests on.

Reply:
Thanks for clarifying that, I hadn't noticed it. Is that file also listing the sets of options used for the fixtures that Ollie was mentionning this morning?

Reply:
Yup, those are used to generate the fixtures during certain Gulp tasks: https://github.com/alphagov/govuk-frontend/blob/54272096c11bf73c6e1fa5dd992e01c5e42a8a93/tasks/gulp/copy-to-destination.js#L56

Reply:
Cheers, I'll add that to the descrption as well, then.